Resources/Prototypes/_Impstation/Entities/Objects/Devices/flatpack.yml

```diff
@@ -21,3 +21,12 @@
   components:
   - type: Flatpack
     entity: ChemMaster
+
+- type: entity
+  parent: BaseFlatpack
+  id: WirelessCameraMovableFlatpack
```

Comment:
this isn't actually used anywhere, was it supposed to be included?

Reply:
Oh shit I forgot I made that while I was there. Ill cut that out